README.md

```diff
@@ -366,6 +366,19 @@ UUID will be added to each event record automatically if you call `enableAutoApp
 
 It outputs the value as a column name `record_uuid` by default.
 
+### Adding Advertising Id to each event record automatically
+Advertising Id will be added to each event record automatically if you call `enableAutoAppendAdvertisingIdentifier`. You must install Google Play Service Ads (Gradle com.google.android.gms:play-services-ads) as an dependency for this feature to work. User must also not turn on Limit Ad Tracking feature in their device, otherwise Treasure Data will not attach advertising id to the record.
```

Comment:
Advertising Id will be added to each event record automatically if you call `enableAutoAppendAdvertisingIdentifier`. You must install Google Play Service Ads (Gradle com.google.android.gms:play-services-ads) as an dependency for this feature to work. User must also not turn on Limit Ad Tracking feature in their device, otherwise Treasure Data will not attach advertising id to the record.
```suggestion
Advertising Id will be added to each event record automatically if you call `enableAutoAppendAdvertisingIdentifier`. You must install Google Play Service Ads (Gradle `com.google.android.gms:play-services-ads`) as a dependency for this feature to work. User must also not turn on Limit Ad Tracking feature in their device, otherwise, Treasure Data will not attach advertising ID to the record.
```